mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -148,6 +148,8 @@ the provider should wait for a single HTTP request.  This will not adjust the
 amount of time the provider will wait for a logical operation - use the resource
 timeout blocks for that.
 
+* `request_reason` - (Optional) Send a Request Reason [System Parameter](https://cloud.google.com/apis/docs/system-parameters) for each API call made by the provider.  The `X-Goog-Request-Reason` header value is used to provide a user-supplied justification into GCP AuditLogs. The provider configuration setting overrides any value set using the `CLOUDSDK_CORE_REQUEST_REASON` environment variable.
```

Comment:
```suggestion
* `request_reason` - (Optional) Send a Request Reason [System Parameter](https://cloud.google.com/apis/docs/system-parameters) for each API call made by the provider.  The `X-Goog-Request-Reason` header value is used to provide a user-supplied justification into GCP AuditLogs.
```

Reply:
Most examples only include an environment variable below-the-fold.

Reply:
done